pkg/boot/linux/load_linux_arm64.go

```diff
@@ -0,0 +1,235 @@
+// Copyright 2022 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build (linux && arm64) || (linux && riscv64)
+// +build linux,arm64 linux,riscv64
+
+package linux
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"os"
+	"syscall"
+
+	"github.com/u-root/u-root/pkg/boot/image"
+	"github.com/u-root/u-root/pkg/dt"
+	"github.com/u-root/u-root/pkg/uio"
+	"golang.org/x/sys/unix"
+)
+
+// loadFDT loads a flateen device tree blob.
+//
+// It first tries to load it from given file path, then
+// from /sys/firmware/fdt. If still failed, it resorts
+// to read /proc/device-tree, and flatten it.
+func loadFDT(dtb string) (*dt.FDT, error) {
```

Comment:
If it is not implemented, does it need to be here? this is confusing in such a big PR.

Reply:
pkg/dt seems a better place. Implementing it.

Reply:
Done.